tools/clang/test/CodeGenHLSL/vk-namespace.hlsl

```diff
@@ -0,0 +1,21 @@
+// RUN: %dxc -E main -T vs_6_0 %s | FileCheck %s
+
+struct SInstanceData {
+  float4x3 VisualToWorld;
+  float4 Output;
+};
+
+struct VS_INPUT	{
+  float3 Position : POSITION;
+  SInstanceData	InstanceData : TEXCOORD4;
+};
+
+float4 main(const VS_INPUT v) : SV_Position {
+	const SInstanceData	I = v.InstanceData;
+  uint64_t clock;
+// CHECK: error: use of undeclared identifier 'vk'
+  clock = vk::ReadClock(vk::DeviceScope);
+// CHECK: error: use of undeclared identifier 'vk'
+  clock = vk::ReadClock(vk::SubgroupScope);
+  return I.Output;
+}
```

Comment:
The test looks good! We've moved almost all such shaders from CodeGenHLSL to HLSLFilecheck. I would recommend this directory: https://github.com/microsoft/DirectXShaderCompiler/tree/master/tools/clang/test/HLSLFileCheck/hlsl/namespace

Reply:
Done